pkg/syncer/admissionhandler/validatepvcannotation.go

```diff
@@ -84,3 +89,51 @@ func validatePVCAnnotation(ctx context.Context, request admission.Request) admis
 	log.Debugf("validatePVCAnnotation completed for the request %v", request)
 	return admission.Allowed("")
 }
+
+// Disallow editing PVC Volume Health annotation for non cns-csi service account users.
+func validatePVCVolumeHealthAnnotation(ctx context.Context, request admission.Request) admission.Response {
```

Comment:
I would suggest to start a new file for the validation handler for volume health case

Also, naming wise, I would suggest something like, `validatePVCAnnotationForVolumeHealth`. Likewise, the existing `validatePVCAnnotation` should have been named as `validatePVCAnnotationForTKGSHA` though.

Reply:
Why we need a new file for this? I think we can put all code related to pvc validation in one file. But I agree that we need to rename  the existing validatePVCAnnotation should have been named as validatePVCAnnotationForTKGSHA

Reply:
My thinking here is to make each admissionhandler file single-purposed. If we keep growing the file with different validation handling functions for PVC annotation, it could possibly make this file extremely long and less readable some day. Just a suggestion. Please think about it. I am ok with either way though.

Reply:
I've went with separate files. They are all in the same package so it's an easy reversible change if we want to consolidate some handlers in the future.